components/brave_wallet_ui/page/screens/send/components/account-selector/account-selector.tsx

```diff
@@ -75,6 +77,14 @@ export const AccountSelector = (props: Props) => {
       return []
     }
 
+    if (selectedNetwork.coin === BraveWallet.CoinType.FIL) {
+      let filecoinAccounts = accounts.filter((account) =>
```

Comment:
Should we also add a condition for `selectedNetwork.coin`? There's no _actual_ guarantee that `selectedNetwork` and `selectedAccountId` are in sync.

Reply:
I've moved from using selectedNetwork.coin to selectedAccountId.coin